crates/neon-sys/build.rs

```diff
@@ -164,18 +164,19 @@ mod build {
         }
 
         if cfg!(windows) {
-            let node_gyp_output = String::from_utf8_lossy(&output.stderr);
-            println!("cargo:node_arch={}", parse_node_arch(&node_gyp_output));
```

Comment:
`node_gyp_output` no longer exists. Does it need to be passed into this method?

Reply:
Oops - that line shouldn't have been deleted - fixed. (For complex dependency reasons I had to backport the fix to 0.3.x to test it and had to manually merge as the indenting level had changed, so must have missed this one).